.github/workflows/build-and-test.yml

```diff
@@ -109,6 +109,10 @@ jobs:
       run: npm run build
       working-directory: gitgitgadget
 
+    - name: Run linter
```

Comment:
The `test` script in package.json already runs lint.  Do we want a script that only runs jest?  I don't always appreciate waiting for linting to run when the change is minor.  Having the CI workflow really reduces the need to run linting with each test run.  The VS Code ESLint extension gets the linting done in real time.

If  you update the yml, could you move this step before the build step?

Reply:
Makes sense!

Reply:
Fixed.